### ISSUE TITLE
add title attribute on iframe

### ISSUE THREAD

**Issue by muhammad-ammar** [masenius/tableau-react-embed#6]
For[ iframe accessibility](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#accessibility_concerns), it is recommended to add `title` attribute on `iframe`. But I am unable to find an option to add `title` on iframe added by `TableauReport`. Need help on how can add title attribute in `TableauReport`.

**Comment by muhammad-ammar** [comment]
@masenius any help?
